src/web/src/components/Organisation/Upsert/OrgRolesEdit.tsx

```diff
@@ -215,7 +215,7 @@ export const OrgRolesEdit: React.FC<InputProps> = ({
 
   const form = useForm({
     mode: "all",
-    // resolver: zodResolver(schema),
```

Comment:
how did that happen? 🤦‍♂️

Reply:
My bad, commented out during testing 🤦‍♂️🤦‍♂️🤦‍♂️